QStore4S/src/main/java/edu/asu/qstore4s/controller/QStore.java

```diff
@@ -36,21 +35,20 @@
 @Controller
 public class QStore {
 
-    private static final Logger logger = LoggerFactory.getLogger(QStore.class);
     private static final String RETURN_JSON = "application/json";
 
     private static final String XML = "application/xml";
     private static final String JSON = "application/json";
 
-    @Autowired
-    IRepositoryManager repositorymanager;
-
-    @Autowired
-    @Qualifier("neo4jOperations")
-    private Neo4jOperations template;
+    private static final HashMap<String, Class<?>> classMap = new HashMap<String, Class<?>>() {
+        {
+            put("relationevent", RelationEvent.class);
+            put("appellationevent", AppellationEvent.class);
```

Comment:
should also not be static. There is simply no reason here why that would have to be static, and if anything only makes it more difficult to maintain.